.pre-commit-config.yaml

```
@@ -0,0 +1,79 @@
+repos:
+  - repo: https://github.com/nbQA-dev/nbQA
+    rev: 1.8.5
+    hooks:
+      - id: nbqa-ruff
+        args:
+          - --fix
+          - --target-version=py311
+          - --ignore=E721,E722
+          - --line-length=80
+      - id: nbqa-black
+        args:
+          - --line-length=80
```

```suggestion
          - --line-length=160
```
this is a pet peeve of mine

---

According to Python documentation, 79/80 should be the goal: https://peps.python.org/pep-0008/#maximum-line-length

It's certainly your repo, but 80 is the standard.